pkgs/top-level/config.nix

```
@@ -131,10 +143,26 @@ let
     checkMeta = mkOption {
       type = types.bool;
       default = false;
-      description = ''
+      description = lib.mdDoc ''
         Whether to check that the `meta` attribute of derivations are correct during evaluation time.
       '';
     };
+
+    packageOverrides = mkOption {
+      default = const {};
+      type = overrideFnType;
+      description = lib.mdDoc ''
+        **Deprecated.** Use `overlays` instead!
```

Deprecation hints at removal, which has not been decided, and would seem unproductive to me.

```suggestion
        **Not recommended.** Use `overlays` instead.
```

---

It was worth a try ;-)